### PR TITLE
fix(reducer): rename actions by naming conventions

### PR DESCRIPTION
**Pull request by maximilianoforlenza**
fix(reducer): fix when fetch current household

### CODE REVIEW

File: src/reducers/survey.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ import {
     CLOSE_DWELLING_VISIT_SUCCEEDED,
     CLOSE_HOUSEHOLD_VISIT_REQUESTED,
     CLOSE_HOUSEHOLD_VISIT_SUCCEEDED,
-    CURRENT_HOUSEHOLD_VISIT_REQUESTED,
+    FETCH_CURRENT_HOUSEHOLD_VISIT_REQUESTED,
```
Comment (navarroaxel) [inline src/reducers/survey.js]: naming convention `<entity>_<verb>_<action>(past)`

**Comment by maximilianoforlenza** [comment]
listo

File: src/actions/index.js [hunk 2]
```diff
@@ -3,14 +3,14 @@ import {
...
-    CLOSE_HOUSEHOLD_VISIT_REQUESTED,
+    DWELLING_CLOSE_VISIT_REQUESTED,
+    HOUSEHOLD_CLOSE_VISIT_REQUESTED,
     DWELLING_FETCH_REQUESTED,
     DWELLING_UPDATE_REQUESTED,
     DWELLINGS_FETCH_REQUESTED,
-    FETCH_DWELLING_VISITS_REQUESTED,
-    FETCH_CURRENT_DWELLING_VISIT_REQUESTED,
-    FETCH_CURRENT_HOUSEHOLD_VISIT_REQUESTED,
+    DWELLING_FETCH_VISITS_REQUESTED,
```
Comment (navarroaxel) [inline src/actions/index.js]: DWELLING_VISITS_FETCH_REQUESTED

**Comment by maximilianoforlenza** [comment]
listo

**Pull request merged**